### ISSUE TITLE
yolov3.yaml if classes removed, lower performance

### ISSUE THREAD

**Issue by chbloca** [leggedrobotics/darknet_ros#158]
Since I would like to use this package just for detecting certain object categories. I just delete from the list in yolov3.yaml the objects I do not want to appear. However, I have notice this does not work as it should since the YOLO will anyways make detections on such objects but will label them with the survivals on the list.

In addition, the quality of detection of the surviving objects in the list decreases unexplicably.

For instance, if I just want to detect persons, there will be a lot of Positive Negatives.
![IMG_20190507_144027](https://user-images.githubusercontent.com/45597426/57384359-224ef880-71b9-11e9-93a2-c5961bc6f52e.jpg)

**Comment by rekalantar** [comment]
I think you meant false positives?

**Comment by chbloca** [comment]
> 
> 
> I think you meant false positives?

No, True Negatives, it is different stuff. See the picture above.

**Comment by fiorano10** [comment]
@chbloca You can't just delete labels...you need to retrain the model for only those objects, look at @AlexeyAB repo.

**Comment by 9527-csroad** [comment]
If you don't want to retrain a model and only want to detect one class or several classes. Maybe you can try this method #204 .
